src/modules/getData.js

```diff
@@ -0,0 +1,22 @@
+const gameUser = document.getElementById('game-user');
+const gameScore = document.getElementById('game-score');
+const gameScoreBtn = document.getElementById('score-btn');
+
+const myGameUSerData = async () => {
+  await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/LoviMadaan1209aaradhya/scores',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: gameUser.value,
+        score: gameScore.value,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    },
+  )
+    .then((response) => response.json());
```

Comment:
[OPTIONAL] You did a lovely job :clap: but keep in mind whenever you use `async` and `await` please don't use `.then` and also use `DRY` principles . "Don't repeat yourself" (DRY) is a principle of software development aimed at reducing the repetition of software patterns, replacing it with abstractions or using data normalization to avoid redundancy. check my suggestions below :point_down: 

```suggestion
 const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/LoviMadaan1209aaradhya/scores';
  const data = {
    user: gameUser.value,
    score: gameScore.value,
  };
  const options = {
    method: 'POST',
    body: JSON.stringify(data),
    headers: {
      'Content-type': 'application/json; charset=UTF-8',
    },
  };

  const response = await fetch(url, options);
  const jsonResponse = await response.json();

  return jsonResponse;
```